packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -470,6 +470,47 @@ describe('c-quantic-search-box-input', () => {
               expectedFirstSuggestionSelected
             );
           });
+
+          it('should display the clear input icon', async () => {
+            const element = createTestComponent({
+              ...defaultOptions,
+              suggestions: mockSuggestions,
+              textarea: textareaValue,
+            });
+            setupEventListeners(element);
+            await flushPromises();
+
+            const input = element.shadowRoot.querySelector(
+              textareaValue
+                ? selectors.searchBoxTextArea
+                : selectors.searchBoxInput
+            );
+            expect(input).not.toBeNull();
+
+            await input.focus();
+
+            const suggestionsList = element.shadowRoot.querySelector(
+              selectors.searchBoxSuggestionsList
+            );
+            expect(suggestionsList).not.toBeNull();
+
+            const querySuggestionIndex = 0;
+            const firstSuggestion = suggestionsList.shadowRoot.querySelectorAll(
+              selectors.suggestionOption
+            )[querySuggestionIndex];
+            expect(firstSuggestion).not.toBeNull();
+
+            firstSuggestion.dispatchEvent(new CustomEvent('mousedown'));
+
+            expect(
+              functionsMocks.exampleSelectSuggestion
+            ).toHaveBeenCalledTimes(1);
```

Comment:
As the issue is not related to selecting from suggestions list, I don't think the test should contain this.

Please make a simple test that just checks for the presence of clear icon when this component renders and when  a value is passed to `inputValue`.
This simple test should be failing before applying the fix to the `quanticSearchBoxInput.js` file and should pass after adding the fix.